libraries/core-react/src/components/TextField/TextField.test.tsx

```diff
@@ -50,6 +54,40 @@ describe('TextField', () => {
     expect(value).toBe(newValue)
   })
 
+  it('Can have a helper text', () => {
+    const helperText = 'Help me'
+
+    render(<TextField id="test" helperText={helperText} />)
+    const inputNode = screen.getByText(helperText)
+    expect(inputNode).toBeDefined()
+  })
+
+  it('Can have an icon inside the input', () => {
+    const labelText = 'Label for test'
+    render(
+      <TextField
+        id="test"
+        label={labelText}
+        inputIcon={<Icon name="save" />}
+      />,
+    )
+    const inputNode = screen.getByLabelText(labelText).parentNode
+    expect(inputNode.querySelectorAll('svg')).toHaveLength(1)
```

Comment:
Suggest adding `title` and assertion on this as we are using that in the stories examples

Reply:
I copied that Icon from Accordion and we use `title` there. I'm not sure what we actually recommends here? Is it for a11y? If so, wouldn't it be better to  use aria-label on the icon instead of title? Or in combination? Maybe we should highlight several a11y options in the storybook? 🤔  I'm not sure if it think it's worth testing for, should probably be tested in the Icon component @mimarz

Reply:
Not sure if its needed when its _inside_ the `TextField`, but its recommended a11y when the svg/icon is [standalone](https://css-tricks.com/accessible-svgs/). We are using the actual `<title>` element inside here.

Reply:
@wenche maybe remove the title from the stories then?

Reply:
I think we should either use aria-label="something" or aria-hidden={true} @mimarz

Reply:
How do you suggest we do this? [Because we are using it inside Icon already](https://github.com/equinor/design-system/blob/31eac3c103362dd416ffe58d62e0175f2fca7e09/libraries/core-react/src/components/Icon/Icon.tsx#L111)

Reply:
<Label>'s id would probably be best to set as `aria-labelledby` on `<Icon>` in this use-case maybe (given title is not set)? @wenche